src/EnergyPlus/General.cc

```diff
@@ -797,13 +763,11 @@ std::string CreateSysTimeIntervalString(EnergyPlusData &state)
     if (TimeStmpE[3] == ' ') {
         TimeStmpE[3] = '0';
     }
-    OutputString = TimeStmpS + " - " + TimeStmpE;
-
-    return OutputString;
+    return TimeStmpS + " - " + TimeStmpE;
 }
 
 // returns the Julian date for the first, second, etc. day of week for a given month
-int nthDayOfWeekOfMonth(EnergyPlusData &state,
+int nthDayOfWeekOfMonth(const EnergyPlusData &state,
```

Comment:
[src/EnergyPlus/General.cc:806]:(style),[constParameter],Parameter 'state' can be declared as reference to const